### ISSUE TITLE
training on reconstructed images

### ISSUE THREAD

**Issue by zhangxin-xd** [magic-research/Dataset_Quantization#17]
Thank you for your excellent work!

I encountered some issues while reproducing the CIFAR-10 results reported in your paper. Specifically, I observed the following:

When training the coreset on the original dataset (with a data keep ratio of 10%), the performance reached 84% accuracy. However, when using the reconstructed images under the same setting, the performance dropped to 73% accuracy.

This discrepancy seems unusual, and I would greatly appreciate your guidance on resolving it.

For your reference, here is the script I used for the reconstruction process:

`CUDA_VISIBLE_DEVICES=0 python -u quantize_pixel.py \
    --data CIFAR10 --data_path data_cifar \
    --output_dir results/pixel_quantized_cifar --model mae_vit_large_patch16 \
    --resume pretrained/mae_visualize_vit_large_ganloss.pth --batch_size 128 \
    --mask_ratio 0.2 --cam_mask`

![Image](https://github.com/user-attachments/assets/731f74c2-9b6f-43d7-a27d-01b51a72ff42)

I’m looking forward to your insights and any suggestions you may have.

**Comment by vimar-gu** [comment]
Thanks for your interest in our work!

Firstly, it should be noted that the reconstruction further saves 20% of storage space from using the original ones. So to validate the performance of 10% data keep ratio, you need to use 12.5% of the sample number. 

The larger performance drop does seem strange to me. Can you try reconstruction and validation both with different random seeds and multiple runs?

**Comment by zhangxin-xd** [comment]
Thank you for your prompt response. The issue has been resolved—it appears to have been an environment-related problem.
